timing_bench_test.go

```diff
@@ -46,7 +47,9 @@ func BenchmarkChannel(b *testing.B) {
 	b.ResetTimer()
 	for i := 0; i < concurrency; i++ {
 		go func() {
-			ch <- &timing{host: "myhost", dur: dur}
+			for j := 0; j < int(math.Ceil(float64(b.N)/concurrency)); j++ {
```

Comment:
If I make the test channel a buffered channel with depth 1000, the time to submit goes up slightly but is still tiny. This tells me that a buffered channel is probably worth doing here.

```
➜  httpseverywhere git:(simpler) ✗ go test -v -run blah -bench . | grep -v "^DEBUG"
BenchmarkNoMatch-4   	DEBUG httpse: httpse.go:156 Max running time: 0ms for host:
   30000	     51011 ns/op
BenchmarkChannel-4   	 2000000	       774 ns/op
BenchmarkLock-4      	DEBUG channel: timing_bench_test.go:32 Average running time: 0ms
   50000	     36920 ns/op
PASS
ok  	github.com/getlantern/httpseverywhere	12.619s
```